src/rt.rs

```diff
@@ -53,4 +55,14 @@ impl Rt {
                 .await
         })
     }
+
+    pub(crate) fn cancel_tasks(&mut self) {
+        _ = mem::replace(&mut self.local, new_local());
```

Comment:
This is a nice idea. I was messing around with abort + await at some point and it's annoying/unrealistic to crash like that.